services/app-api/src/authn/cognitoAuthn.ts

```diff
@@ -227,26 +187,19 @@ export async function userFromCognitoAuthProvider(
 
         // if it is a state user, insert the state they are from
         if (cognitoUser.role === 'STATE_USER') {
-            userToInsert.stateCode = cognitoUser.state_code
+            userToInsert.stateCode = cognitoUser.stateCode
         }
 
-        try {
-            const result = await store.insertUser(userToInsert)
-            if (isStoreError(result)) {
-                console.error(
-                    `Could not insert user: ${JSON.stringify(result)}`
-                )
-                return cognitoUserResult
-            }
-            return userTypeFromUser(result)
-        } catch (e) {
-            console.error(`Could not insert user: ${JSON.stringify(e)}`)
+        const result = await store.insertUser(userToInsert)
```

Comment:
🔴 I think we need the `try...catch` still here right-- and then inside the catch use `convertPrismaErrorToStoreError`.  Also we should log to OTEL.

I don't think isStoreError covers things that seems like something different peeking at the code